fetch.bs

```diff
@@ -5685,7 +5686,14 @@ that RFC's normative processing requirements. [[RFC2397]]
 
  <li><p>Set <var>mimeTypeBytes</var> to the <a>isomorphic encode</a> of <var>mimeType</var>.
 
- <li><p>Return <var>mimeTypeBytes</var> and <var>body</var>.
+ <li><p>Let <var>dataURLStruct</var> be a <a>struct</a> consisting of a
```

Comment:
We'd need to add indexing syntax to Infra to be able to address tuple entries then. Currently you have to name them.